### PR TITLE
Fix link in contribution guide

### PR DESCRIPTION
**Pull request by mayocream**
I think the "Members section in `README.md`", which means the `README.md` file in the root.

### CODE REVIEW

File: CONTRIBUTING/first-time-contributions.md [hunk 1]
```diff
@@ -60,4 +60,4 @@ Our members list can be found
 
 Once you have successfully get your
 first PR merged, you can add your name
-to our Members section in [README.md](README.md).
+to our Members section in [README.md](https://github.com/cncf/tag-security/blob/main/README.md).
```
Comment (TheFoxAtWork) [inline CONTRIBUTING/first-time-contributions.md:63]:
```suggestion
to our Members section in [README.md](https://github.com/cncf/tag-security#members).
```

Comment (TheFoxAtWork) [reply CONTRIBUTING/first-time-contributions.md:63]: @mayocream one small change to correct it to the Members section.

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
small change requested

**Comment by mayocream** [comment]
nice :)

**Review by TheFoxAtWork — APPROVED** [review]
LGTM

**Pull request merged**